packages/schematics/update/update/index.ts

```diff
@@ -210,7 +216,8 @@ function _validateUpdatePackages(
     logger.debug(`${name}...`);
 
     const peers = target.packageJson.peerDependencies || {};
```

Comment:
Nit can be simplified by
`const {peerDependencies = {}, peerDependenciesMeta = {}} = target.packageJson`

Reply:
Fixed